README.rst

```diff
@@ -1,23 +1,150 @@
-kinto2xml
----------
+#########
+amo2kinto
+#########
 
-Generate a blocklist.xml file from Kinto collections.
+
+Installation
+============
+
+To install the release, just use::
```

Comment:
nit: «just use» is superflous, we know that the following is a command to use/run :)